package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "watchTests": "jest --watch",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
- [ ] There's no point adding a script if that doesn't work properly.
Please either follow the instructions on the [webpack documentation ](https://webpack.js.org/guides/development/#using-webpack-dev-server)to make the dev server script work or remove it from the scripts list.
  - The awesome thing about the `npm start` script is that it will open the compiled code on your browser automatically when properly configured and update without need to run build again, it's super useful while coding.

Reply:
- [ ] Just as mentioned by the last reviewer, the `npm start` command is still not working. See the error message below:
![cast](https://user-images.githubusercontent.com/26856491/115433470-841a9880-a1ff-11eb-98bf-2d0f67b0e3fd.png)

Reply:
As mentioned by the last reviewer, one of the scripts `npm start` is still not working. Kindly check the error message below:
![cast](https://user-images.githubusercontent.com/26856491/115433701-bd530880-a1ff-11eb-9d83-2e53e320b9a6.png)